src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: OpenTelemetry APM UI
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: The OpenTelemetry APM UI provides tools for identifying and diagnosing problems with services monitoring with OpenTelemetry.
+freshnessValidatedDate: never
+redirects:
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-view-your-data
+  - /docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-view-your-data
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-find-entities
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-distributed-tracing-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-transactions-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-database-page
+  - docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-external-services-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-jvms-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-go-runtime-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-logs-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-errors-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-errors-inbox-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-metrics-explorer-page
+---
+
+import opentelemetryViewSpanEvents from 'images/opentelemetry_screenshot-crop_view-span-events.webp'
+
+The OpenTelemetry APM UI is a curated experience available for [service entities][service entity attributes]. See [set up your own app](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app) for details on configuring services with OpenTelemetry APM monitoring. See [Find OpenTelemetry APM services](#find-opentelemetry-apm-services) to find OpenTelemetry APM entities in New Relic.
```

Comment:
@jack-berg Where is "[service entity attributes]" supposed to link to?

Reply:
This is a usage of the referenced link declared at the bottom of the document.